scripts/b/bootstrap.py

```diff
@@ -4,13 +4,29 @@
 import shutil
 import subprocess
 import sys
+import os
 
+VENV_PATH = '.venv'
 
 def check_version():
     version = sys.version_info
-    if version.major != 3 or version.minor < 8:
-        raise Exception("This script requires Python 3.8 or higher.")
+    if version.major != 3 or version.minor < 8 or version.minor > 10:
+        raise Exception("This script requires Python >=3.8,<=3.10")
+
+def check_venv_status(venv_path):
```

Comment:
Ahh, nice! Didn't realize the build system was already creating `.venv` and makes sense we don't want to bloat the other`./b`commands that are much shorter. 

re git lfs: Oh interesting. When I did a git clone, it didn't automatically git lfs checkout, but agreed since it should only be a one time thing anyways, it doesn't need to be in the run script. 

Changes look good to me! Appreciate the quick reply.

Closing the PR now